external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/CachedKafkaConsumer.scala

```diff
@@ -398,4 +386,23 @@ private[kafka010] object CachedKafkaConsumer extends Logging {
       consumer
     }
   }
+
+  private def reportDataLoss0(
```

Comment:
The `additionalInfo` requires passing a lot more variables. I can pass all of it, but yeah